docs/index.rst

```diff
@@ -1,5 +1,5 @@
-TUF Developer Documentation
-===========================
+Python-TUF |version| Developer Documentation
+=======================================================================
```

Comment:
```suggestion
Python-TUF |version| Developer Documentation
============================================
```

Reply:
I could have mentioned this: The issue is that the line needs to be at least as long as the title -- and we don't know how long the title is because of the variable expansion.

Luckily it's OK for the line to be longer